src/test/java/cart/domain/member/MemberTest.java

```diff
@@ -0,0 +1,24 @@
+package cart.domain.member;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class MemberTest {
+    private static final String EMAIL = "glen@naver.com";
+    private static final String PASSWORD = "123456";
+
+    @Test
+    @DisplayName("사용자가 정상적으로 생성되어야 한다.")
+    void create_success() {
+        // given
+        Member member = new Member(new Email(EMAIL), new Password(PASSWORD));
+
+        // expect
+        assertThat(member.getEmail())
+                .isEqualTo("glen@naver.com");
+        assertThat(member.getPassword())
+                .isEqualTo("123456");
+    }
```

Comment:
```suggestion
        // expect
        assertThat(member.getEmail()).isEqualTo(EMAIL);
        assertThat(member.getPassword()).isEqualTo(PASSWORD);
    }
```

검증시 요청에서 보낸 값을 사용하면 더 정확할 것 같아요!

Reply:
[검증부 (assert / expect)는 하드코딩한다](https://jojoldu.tistory.com/615)
라는 글을 본 적이 있어서, 최대한 검증부 쪽은 값을 수동으로 적는 것을 선호하는 편입니다!

Reply:
해당 글에서는 검증부에 도메인 로직을 이용하는 것에 대한 이야기를 하고 있네요!
제가 제안드린건 하드코딩된 값을 이용하되, 가독성을 위해 변수로 빼서 사용하자였습니다. :) 
따라서 프로덕션 코드와 강결합이 되는 등 해당 글에서 말하는 단점이 적용될 것 같진 않네요. 
이 부분은 글렌이 원하시는대로 해주세요~